internal/cli/backup/delete.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright (c) 2024 OceanBase
+ob-operator is licensed under Mulan PSL v2.
+You can use this software according to the terms and conditions of the Mulan PSL v2.
+You may obtain a copy of Mulan PSL v2 at:
+
+	http://license.coscl.org.cn/MulanPSL2
+
+THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+See the Mulan PSL v2 for more details.
+*/
+package backup
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/oceanbase/ob-operator/internal/cli/generic"
+	"github.com/oceanbase/ob-operator/internal/clients"
+)
+
+type DeleteOptions struct {
+	generic.ResourceOption
+	force bool
+}
+
+func NewDeleteOptions() *DeleteOptions {
+	return &DeleteOptions{}
+}
+
+func DeleteTenantBackupPolicy(ctx context.Context, o *DeleteOptions) error {
+	nn := types.NamespacedName{Name: o.Name, Namespace: o.Namespace}
```

Comment:
This is incorrect. This should accept tenant name, and take it as label to filter existing policy and delete it.